server/public/model/config.go

```diff
@@ -4672,7 +4674,13 @@ func (o *Config) GetSanitizeOptions() map[string]bool {
 	return options
 }
 
-func (o *Config) Sanitize(pluginManifests []*Manifest) {
+// Sanitize removes sensitive information from the configuration object.
+// It replaces sensitive fields with [FakeSetting] or sanitizes them.
+//
+// Parameters:
+//   - pluginManifests: Plugin manifests for sanitizing plugin settings.
+//   - partiallySanitizeDataSources: Whether to partially sanitize SQL data sources instead of fully replacing it with [FakeSetting].
```

Comment:
Maybe we should be a bit more explicit about the differences of this parameter, ideally with an example. Or referring to the docs of `SanitizeDataSource`.

Reply:
I've aded some comments to `SanitizeOptions` .